docs/extensions/tutorials/_posts/1970-01-09-FAQ.md

```diff
@@ -63,35 +60,4 @@ Yes, you can use them the [same](https://facebook.github.io/react-native/docs/na
 
 ## What do I need to run the app locally?
 
-There are 2 ways you can run your app locally:
-
-- 1) Using **Shoutem Preview** app (available for [iOS]({{ site.shoutem.previewAppiOS }}) and [Android]({{ site.shoutem.previewAppAndroid }}))
-- 2) Running app on the iOS/Android simulators or directly on devices
-
-For both cases, you will need [Node.js v7](https://nodejs.org/en/) and [react-native-cli](https://www.npmjs.com/package/react-native-cli).
-
-### Node.js
-
-Install Node.js from their website. If you already have it installed, check which version you have with  `node -v` command. If the version is not in `v7.*.*` range, we recommend using [nvm](https://github.com/creationix/nvm) to upgrade it.
-
-### react-native-cli
-
-Install the `react-native-cli` globally with npm:
-
-```ShellSession
-$ npm install -g react-native-cli
-```
-
-If the previous commands fails because of the permission issues, run it with `sudo` prefix: `sudo npm install -g react-native-cli`.
-
-##### 1) Using Shoutem Preview app
-
-For this, you only need Node.js v7, `react-native-cli` and `@shoutem/cli`! Use `shoutem run` command.
-
-##### 2) Running app on the iOS/Android simulators or directly on devices
-
-Using Shoutem Preview app, you can run your app if custom developed extensions don't include the native code (since our binary doesn't have it). If you have native dependencies, setup the React Native local environment. Read more about this in [local environment setup tutorial]({{ site.url }}/docs/extensions/tutorials/setting-local-environment).
-
-## Can I debug using Shoutem Preview app?
-
-This feature is coming very soon!
+Since Shoutem apps are React Native apps, you need to [set up your local environment]({{ site.url }}/docs/extensions/tutorials/setting-local-environment) the same way you'd do it for a React Native app.
```

Comment:
After that use Shoutem CLI tool to get a local copy of your app using `clone` command.